pkg/scheduler/framework/runtime/framework.go

```diff
@@ -1038,7 +1038,7 @@ func (f *frameworkImpl) RunBindPlugins(ctx context.Context, state *framework.Cyc
 	}
 	for _, bp := range f.bindPlugins {
 		status = f.runBindPlugin(ctx, bp, state, pod, nodeName)
-		if status != nil && status.Code() == framework.Skip {
+		if status.Code() == framework.Skip {
```

Comment:
Add a member function for `status.IsSkip()` ?

Reply:
This is the only place we used to verify the Skip code, so I prefer leave it is. And maybe add `IsSkip` method one day we have redundant codes.

Reply:
not for reducing the redundant codes. We have `IsSuccess` `IsUnschedulable` `IsWait`. Add `IsSkip` is to ensure consistent code formatting.

Reply:
sg.

Reply:
> Add a member function for status.IsSkip() ?

I recalled recently there is an issue discussing this, but cannot find the issue number..